controllers/userController.js

```diff
@@ -41,6 +45,7 @@ module.exports = {
         })
         .catch(() => res.status(500).send({ error: 'Internal server error' }));
     } catch (err) {
+      console.log(err);
```

Comment:
Hi. Could you take out this console log? We might also enable the linter to flag future log statements.

Reply:
Okay, I'll get on this right now.